app/error_handlers.py

```diff
@@ -30,4 +30,12 @@ def method_not_allowed(e):
 
 @app.errorhandler(500)
 def internal_error(e):
-    return make_response(jsonify(status=500, error='Internal Server Error', message='Well, this is embarrassing...'), status.HTTP_500_INTERNAL_SERVER_ERROR)
+	return make_response(jsonify(status=500, error='Internal Server Error', message='Well, this is embarrassing...'), status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+class InvalidPaymentID(Exception):
+    def __init__(self, message, status_code=None, payload=None):
```

Comment:
Originally, I was passing the entire request back, but I realized that it was unnecessary, will refactor in next commit